packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/quanticFeedbackModal.js

```diff
@@ -0,0 +1,234 @@
+// @ts-nocheck
+import cancel from '@salesforce/label/c.quantic_Cancel';
+import done from '@salesforce/label/c.quantic_Done';
+import explainWhy from '@salesforce/label/c.quantic_ExplainWhy';
+import fillOutField from '@salesforce/label/c.quantic_FillOutField';
+import selectOneOfOptions from '@salesforce/label/c.quantic_SelectOneOfOptions';
+import selectTheReason from '@salesforce/label/c.quantic_SelectTheReason';
+import sendFeedback from '@salesforce/label/c.quantic_SendFeedback';
+import thankYouForYourFeedback from '@salesforce/label/c.quantic_ThankYouForYourFeedback';
+import whatWasTheReason from '@salesforce/label/c.quantic_WhatWasTheReason';
+import yourResponsesHelp from '@salesforce/label/c.quantic_YourResponsesHelp';
+import {getBueno} from 'c/quanticHeadlessLoader';
+import LightningModal from 'lightning/modal';
+import {api} from 'lwc';
+import errorTemplate from './error.html';
+import feedbackFormTemplate from './feedbackForm.html';
+import successTemplate from './success.html';
+
+/**
+ * The `QuanticFeedbackModal` component overlays a message modal on top of the current app window, the modal contains a form that allows the user to give feedback.
+ *
+ * @category Search
+ * @category Insight Panel
+ */
+export default class QuanticFeedbackModal extends LightningModal {
+  /**
+   * @api
+   * The list of options to be displayed in the modal.
+   * @type {Array<{label: string, value: string, withDetails: boolean, detailsRequired: boolean}>}
+   */
+  @api options;
+  /**
+   * @api
+   * The function that will be executed when the feedback is submitted.
+   * @type {function}
+   */
+  @api submitFeedback;
+
+  labels = {
+    done,
+    cancel,
+    sendFeedback,
+    explainWhy,
+    selectOneOfOptions,
+    selectTheReason,
+    whatWasTheReason,
+    fillOutField,
+    yourResponsesHelp,
+    thankYouForYourFeedback,
+  };
+
+  /** @type {string} */
+  feedbackValue;
+  /** @type {string} */
+  detailsValue;
+  /** @type {boolean} */
+  feedbackSubmitted = false;
```

Comment:
```suggestion
  isFeedbackSubmitted = false;
```
Just a suggestion but it makes it clearer that this is a boolean. Also I think the `= false;` is superfluous but I have no problem keeping it since it's clearer as well.